Bottom: 6d251e30f9dc0f8c2be4320137b835b2ff5c867a
Top:    205ed095a64377db5937de2fe85419a26adf3cb2
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-04-29 11:28:04 +0800

Refresh of tun-rework-on-flow-cache

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index d5cca01..aeeb612 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -128,8 +128,6 @@ struct tap_filter {
 #define MAX_TAP_QUEUES 256
 #define TAP_QUEUE_MASK 0xFF
 
-#define TUN_FLOW_EXPIRE (3 * HZ)
-
 /* A tun_file connects an open character device to a tuntap netdevice. It
  * also contains all socket related structures (except sock_fprog and tap_filter)
  * to serve as one transmit queue for tuntap device. The sock_fprog and
